### PR TITLE
playlist delete feature

### PR DESCRIPTION
**Pull request by zaza7**

### CODE REVIEW

File: runtime/internal/voice.js [hunk 1]
```diff
@@ -407,7 +420,7 @@ exports.request = function (msg, suffix, bot) {
               }, Config.settings.deleteTimeout)
             }
           })
-          Logger.error('Playlist failiure, ' + err)
+          Logger.error('Playlist faiiure, ' + err)
```
Comment (LWTechGaming) [inline runtime/internal/voice.js]: You fucked up twice. Pls.

**Comment by LWTechGaming** [comment]
Pushing docs update too.

**Comment by zaza7** [comment]
It was a good idea until we noticed playlist deletion should be level 1.

**Comment by LWTechGaming** [comment]
Lol true

**Pull request closed without merging**